internal/backend/sessions/sessionworkflows/workflow.go

```diff
@@ -574,7 +574,12 @@ func (w *sessionWorkflow) run(wctx workflow.Context, l *zap.Logger) (prints []sd
 			return prints, err
 		}
 
-		if retVal, err = run.Call(ctx, callValue, nil, w.data.Session.Inputs()); err != nil {
+		inputs := map[string]sdktypes.Value{
```

Comment:
Not blocking, but if possible: can you also add `event_type`?

Reply:
in a different pr